plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -382,6 +384,9 @@ open class KtlintPlugin : Plugin<Project> {
 
         val ktlintConfiguration by lazy(LazyThreadSafetyMode.NONE) { createKtlintConfiguration(target, extension) }
         val ktlintRulesetConfiguration = createKtlintRulesetConfiguration(target)
+        val ktlintReporterConfiguration by lazy(LazyThreadSafetyMode.NONE) {
```

Comment:
Isn't that the default thread safety mode?

Reply:
Nope, default is `LazyThreadSafetyMode.SYNCHRONIZED`